helm/hpcc/templates/_helpers.tpl

```diff
@@ -1370,6 +1376,109 @@ spec:
 {{- end }}
 {{- end }}
 
+{{/*
+Builds the commonName for a client certificate.  Used in creation of both certificate and access control list.
+*/}}
+{{- define "hpcc.getClientCommonName" -}}
+  {{- if (.root.Values.certificates | default dict).enabled -}}
```

Comment:
indent: minor.  I think we agreed on using a single space for indenting helm files.  @jakesmith ?

Reply:
yes we did.